.storybook/themes/commerce.scss

```diff
@@ -3,43 +3,543 @@
+				),
+				button-secondary-reverse-hover: (
+					border-color: var(--global-neutral-7),
+					color: var(--global-neutral-7),
+				),
+				button-default: (
+					color: var(--primary-color),
+					background-color: var(--global-white),
+				),
+				button-default-hover: (
+					background-color: var(--global-neutral-1),
+				),
+				carousel: (
+					--slides: 1,
+				),
+				carousel-actions: (
+					display: none,
+					padding-left: var(--global-spacing-4),
+					padding-right: var(--global-spacing-4),
+					//
+					// The width should place the action buttons near the edge of the carousel responsively
+					//
+					// width is derived from adding
+					//   --layout-content-max-width
+					//   2x button widths (--global-font-size-9)
+					//   2x button margins (--global-font-size-5)
+					// then dividing those by
+					//   --layout-max-width
+					// using the formula: (48 + 32 + 1216 + 32 + 48) / 1440
+					//
+					width: calc(calc(var(--layout-content-max-width) + 160) / var(--layout-max-width) * 100%),
+				),
+				carousel-button: (
+					box-shadow: var(--global-box-shadow-1),
+					border-radius: var(--global-spacing-1),
+					background-color: var(--global-white),
+					height: var(--global-spacing-8),
+					width: var(--global-spacing-8),
+				),
+				carousel-track: (
+					gap: var(--global-spacing-2),
+					width: var(--layout-content-scale-width),
+				),
+				date: (
+					font-family: var(--secondary-font-family),
+					font-size: var(--global-font-size-3),
+					line-height: var(--global-line-height-3),
+				),
+				heading: (
+					font-weight: var(--global-font-weight-7),
+				),
+				icon: (
+					fill: var(--global-neutral-8),
+					height: var(--global-spacing-5),
+					width: var(--global-spacing-5),
+				),
+				input: (
+					align-items: flex-start,
+					display: flex,
+					flex-direction: column,
+					gap: var(--global-spacing-1),
+				),
+				input-input: (
+					border: 1px solid var(--global-neutral-5),
+					border-radius: var(--global-border-radius-1),
+					padding: var(--global-spacing-2) var(--global-spacing-3),
+				),
+				input-input-focus: (
+					outline: double 2px var(--global-neutral-5),
+				),
+				input-label: (
+					font-weight: var(--global-font-weight-7),
+				),
+				input-error-input: (
+					border-color: var(--global-red-4),
+				),
+				input-error-input-focus: (
+					outline-color: var(--global-red-4),
+				),
+				input-success-input: (
+					border-color: var(--global-green-5),
+				),
+				input-success-input-focus: (
+					outline-color: var(--global-green-5),
+				),
+				input-warning-input: (
+					border-color: var(--global-orange-2),
+				),
+				input-warning-input-focus: (
+					outline-color: var(--global-orange-2),
+				),
+				link: (
+					color: var(--primary-color),
+				),
+				media-item-fig-caption: (
+					color: var(--global-neutral-5),
+					font-size: var(--global-font-size-2),
+					line-height: var(--global-line-height-2),
+				),
+				pill: (
```

Comment:
In the designs, it looks like there's a success a few other states. This looks new from the designs but we don't have a success pill either 

https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr/Enhanced-Styling-Design-System?node-id=3505%3A27776

Reply:
The design's have been labelled incorrectly here - there is a need for a Pill and a Badge component - at the moment we do not have a "Badge" component that is currently called a Pill in the design document - out "Pill" component is what is being used in the tags block